src/renderer/components/GlobalPage/__tests__/GlobalPage.spec.js

```diff
@@ -0,0 +1,43 @@
+import mockConsole from 'jest-mock-console';
+import { render, screen } from 'testingUtility';
+import { GlobalPage } from '../GlobalPage';
+import { CloudProvider } from '../../../store/CloudProvider';
+import { IpcRenderer } from '../../../IpcRenderer';
+import { CloudStore } from '../../../../store/CloudStore';
+import { themeModes } from '../../theme';
+import * as strings from '../../../../strings';
+
+describe('/renderer/components/GlobalPage/GlobalPage', () => {
+  const extension = {};
+
+  beforeEach(() => {
+    mockConsole(); // automatically restored after each test
+
+    IpcRenderer.createInstance(extension);
+    CloudStore.createInstance().loadExtension(extension);
+  });
+
+  [themeModes.LIGHT, themeModes.DARK].forEach((themeMode) => {
+    describe(`|${themeMode}| mode`, () => {
+      beforeEach(() => {
+        if (themeMode === themeModes.LIGHT) {
+          document.querySelector('body').classList.add('theme-light');
+        } else {
+          document.querySelector('body').classList.remove('theme-light');
+        }
+      });
+
+      it('showing WelcomeView after rendering GlobalPage component', () => {
+        render(
+          <CloudProvider>
+            <GlobalPage />
+          </CloudProvider>
+        );
+
+        expect(
+          screen.getByText(strings.welcome.button.label())
+        ).toBeInTheDocument();
```

Comment:
Instead of just checking it's in the document, let's also check that it's rendered using the light theme vs dark theme:
```suggestion
        expect(
          screen.getByText(strings.welcome.button.label())
        ).toHaveStyleRule('color', 'SOMETHING depending on light or dark mode');
```

Reply:
I did something like ```
expect(screen.getByText(strings.welcome.description())).toHaveStyle(
          'color: var(--textColorPrimary);'
        );
```
But I can't check HEX color, because it told me, that received color in var

Reply:
OK, not the best, but we'll stick with that. Thanks for updating the test!